func.c

```diff
@@ -49,12 +49,12 @@ int login()
     if (strcmp(stored_username_hash, entered_username_hash) == 0 &&
         strcmp(stored_password_hash, entered_password_hash) == 0)
     {
-        printf("[+] Login successful!\n");
+        printf("[+] Login Successful!\n");
         return 1;
     }
     else
     {
-        printf("[!] Login failed. Please check your username and password... :(\n");
+        printf("[!] Login failed. Please check your username and password correctly... :(\n");
```

Comment:
make capital, ALL letters (first letter of the word)

Reply:
done... :)